ogc/docs/maml.md

```diff
@@ -0,0 +1,146 @@
+# MAML: Map Algebra Modeling Language
```

Comment:
Could be useful to have more examples.
We talked about the NRCan slope implementation? 
Is the MAML AST stored as it's own file? You say it can be generated client side. So seems like there are options. 

But if a user just wants to expose a custom slope calculator on their elevation data, how would they do so?